src/System.Net.Http/src/System/Net/Http/HttpContent.cs

```diff
@@ -706,6 +717,9 @@ private void CheckSize(int countToAdd)
             }
         }
 
+#if NET46
+        [SecuritySafeCritical]
```

Comment:
>  .NET Core code has diverged a little bit from the original HttpContent source code in .NET Framework.

Understood. My question is: do we want this in .Net Framework or should it be compiled out using `#if` or getting the .Core separated in a different file? When @karelz and I were discussing the plan, we wanted to ensure 100% app-compat and behavior parity first and add risky enhancements only after proper testing. Do we have CoreFX tests to cover this? What about .Net Framework using NuGet?